src/Tests/Microsoft.NET.Build.Tests/KnownRuntimePackTests.cs

```diff
@@ -139,5 +136,42 @@ private void AddItem(TestAsset testAsset, XElement item)
                 itemGroup.Add(item);
             });
         }
+
+        [WindowsOnlyFact]
+        public void ItCanPublishArm64Winforms()
```

Comment:
You can write unit test for this task like `src\Tasks\Microsoft.NET.Build.Tasks.UnitTests\ProcessFrameworkReferencesTests.cs`. It will be at least better for documenting this long class